DATA/duke.txt

```diff
@@ -0,0 +1,12 @@
+[T][] 1
+[T][X] 1
+[T][] 3
+[T][] 1
+[T][] 2
+[T][] book 
+[T][] book read
+[D][] study book (by: Aug 20 2002)
+[T][] 100
+[D][X] test (by: Feb 22 2022)
+[E][] outing (at: 2pm)
+[T][] 1
```

Comment:
Perhaps including this in `.gitignore` would be a great idea since it's a data text and it's by right a private file.